internal/signalfx-agent/pkg/core/dpfilters/filter.go

```diff
@@ -13,7 +13,7 @@ import (
 type DatapointFilter interface {
 	// Matches takes a datapoint and returns whether it is matched by the
 	// filter
-	Matches(*datapoint.Datapoint) bool
+	Matches(datapointFilter *datapoint.Datapoint) bool
```

Comment:
Similar to the other comment, perhaps simply `datapoint` or `dp` (since it matches the type name).